### PR TITLE
[doc] Support cluster-wide proxy in README

### PR DESCRIPTION
**Pull request by saifshaikh48**

### CODE REVIEW

**Comment by jrvaldes** [comment]
/lgtm

Consider adding the Jira issue to the PR title.

**Comment by saifshaikh48** [comment]
cc: @DCChadwick

File: README.md [hunk 1]
```diff
@@ -221,17 +221,16 @@ Cluster autoscaling is supported for Windows instances.
 Windows instances brought up with WMCO are set up with the containerd container runtime. As WMCO installs and manages the container runtime,
 it is recommended not to preinstall containerd in MachineSet or BYOH Windows instances.
 
+### Cluster-wide proxy 
+WMCO does supports using a [cluster-wide proxy](https://docs.openshift.com/container-platform/latest/networking/enable-cluster-wide-proxy.html)
```
Comment (alinaryan) [inline README.md:225]: supports->support

**Review by alinaryan — COMMENTED** [review]
just a nit:

**Comment by mansikulkarni96** [comment]
I realized this is against 4.14, shouldn't it also go in 4.15?

**Comment by jrvaldes** [comment]
@sebsoto  do you anticipate overriding the e2e job in docs PR? without the skip-if-only-changed I feel they are going to run automatically

**Pull request merged**